src/hotspot/share/classfile/stringTable.cpp

```diff
@@ -241,12 +241,12 @@ void StringTable::create_table() {
 #endif
 }
 
-size_t StringTable::item_added() {
-  return Atomic::add(&_items_count, (size_t)1);
+void StringTable::item_added() {
+  Atomic::inc(&_items_count);
```

Comment:
Can this overflow?

Reply:
Yes, I'm going to revert the _items_count change.